vulnfeeds/cpp/main.go

```diff
@@ -73,21 +75,65 @@ func InScopeGitRepo(repoURL string) bool {
 	return true
 }
 
+// Clones repos and tries to convert version tags to commits
+func GitVersionToCommit(versions cves.VersionInfo, repos []string) (v cves.VersionInfo, e error) {
+	// versions is a VersionInfo with AffectedVersions and no FixCommits
+	// v is a VersionInfo with FixCommits included
+	v = versions
+	for _, repo := range repos {
+		Logger.Infof("Cloning %s", repo)
+		r, err := git.Clone(memory.NewStorage(), nil, &git.CloneOptions{
+			URL: repo,
+		})
+		if err != nil {
+			Logger.Warnf("Failed clone %s: %v", repo, err)
+			continue
+		}
+		for _, av := range versions.AffectedVersions {
+			if av.Fixed == "" {
+				continue
+			}
+			// Need to come up with a way to decide when to prefix
+			ref, err := r.Tag("v" + av.Fixed)
```

Comment:
Let's add a TODO to support some more sophisticated/fuzzy tag -> version matching.

Reply:
Addressed in 7ed688a7d9e25ab364426d06556083b9ffa0229f